jets-bench/src/data_structures.rs

```diff
@@ -169,14 +169,15 @@ impl SimplicityEncode for SimplicityCtx8 {
             .iter()
             .flat_map(|x| x.to_be_bytes())
             .collect::<Vec<u8>>();
-        let mid_state = Value::u256_from_slice(&arr);
+        debug_assert_eq!(arr.len(), 32);
+        let mid_state = Value::power_of_two(&arr);
```

Comment:
In c1527fa7668e0cc423ce5327a222c2cbca1e2485:

I'm confused as to what you're doing here. In the previous version of this commit you used `Value::u256` because this is guaranteed to be 32 bytes. But now you are just adding another runtime assertion. And this still doesn't touch the unnecessary `Vec` in the line above.

Reply:
I can convert `[u32; 8]` directly into `[u8; 32]` if you want. The code will be more verbose because I will have to initialize the array and then assign values byte by byte. I cannot use Rust's iterator API, AFAIK.

I misunderstood your original comment. I thought you called the `Vec` unnecessary because I converted it into an `[u8; 32]` array on the stack. I removed the array, but you seem to have wanted to remove the `Vec` altogether.